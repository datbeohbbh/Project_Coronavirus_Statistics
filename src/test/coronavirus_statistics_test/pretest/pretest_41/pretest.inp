50 100
q 1 35 416
? 4 1
q 1 16 95
q 3 35 190
? 4 2
? 6
q 1 29 330
q 1 48 450
q 2 35 146
q 1 38 276
? 4 2
q 1 19 181
? 4 4
q 3 29 319
? 4 5
q 1 6 18
? 4 1
? 7
? 4 6
? 6
q 3 38 25
? 7
? 4 4
? 4 5
? 6
q 1 40 20
q 3 40 20
q 2 19 17
q 2 35 65
? 4 4
? 4 1
q 3 19 150
q 3 35 4
q 3 29 4
? 6
q 1 32 252
q 1 40 149
? 4 8
q 1 16 73
q 1 37 307
q 1 28 326
q 3 16 142
q 3 32 217
q 3 29 1
q 2 35 5
q 1 31 192
q 2 6 5
q 1 19 339
? 4 6
? 4 9
q 1 2 255
q 1 5 468
q 2 37 194
? 7
q 2 16 19
q 1 21 156
q 3 6 3
? 4 2
q 3 21 35
? 6
q 3 31 169
? 4 3
q 1 42 489
q 1 28 24
q 2 16 6
? 4 4
? 4 1
q 1 36 296
q 1 39 168
? 4 15
q 1 12 335
q 2 31 9
q 1 40 489
? 6
q 1 1 109
q 3 2 131
? 4 16
q 2 1 26
q 1 47 58
? 4 21
q 2 35 4
? 7
? 7
? 7
? 4 21
? 4 9
? 4 11
? 4 2
? 6
? 4 1
? 6
? 4 1
? 4 20
? 4 2
? 6
? 7
? 6
? 4 15
? 4 9
? 4 19