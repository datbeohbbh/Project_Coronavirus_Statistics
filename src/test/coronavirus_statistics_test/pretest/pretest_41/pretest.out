Query ? 4 1 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 416
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 226
-> Has recovered : 0
-> Was dead : 190

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 95
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 276
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 276
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 319

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 319 people

Query ? 4 6 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 276
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 146 people

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 319 people

Query ? 4 4 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 95
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 181
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 146 people

Query ? 4 4 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 95
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 319

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 211 people

Query ? 4 8 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 252
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 149
-> Has recovered : 0
-> Was dead : 20

Query ? 4 9 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 307
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 324 people

Query ? 4 2 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 6
-> Has recovered : 0
-> Was dead : 324

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 216 people

Query ? 4 3 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 7
-> Has recovered : 19
-> Was dead : 142

Query ? 4 4 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 10
-> Has recovered : 5
-> Was dead : 3

Query ? 4 1 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 1
-> Has recovered : 25
-> Was dead : 142

Query ? 4 15 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 353
-> Has recovered : 17
-> Was dead : 150

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 216 people

Query ? 4 16 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 353
-> Has recovered : 17
-> Was dead : 150

Query ? 4 21 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 638
-> Has recovered : 0
-> Was dead : 20

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 324 people

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 324 people

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 324 people

Query ? 4 21 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 638
-> Has recovered : 0
-> Was dead : 20

Query ? 4 9 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 113
-> Has recovered : 194
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 124
-> Has recovered : 0
-> Was dead : 131

Query ? 4 2 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 2
-> Has recovered : 220
-> Was dead : 194

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 220 people

Query ? 4 1 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 1
-> Has recovered : 25
-> Was dead : 142

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 220 people

Query ? 4 1 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 1
-> Has recovered : 25
-> Was dead : 142

Query ? 4 20 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 489
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 2
-> Has recovered : 220
-> Was dead : 194

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 220 people

Query ? 7 :
The city with the highest death is city with the index 29
Quantity: 324 people

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 220 people

Query ? 4 15 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 335
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 113
-> Has recovered : 194
-> Was dead : 0

Query ? 4 19 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 468
-> Has recovered : 0
-> Was dead : 0

